simulator/mips/mips_instr.h

```diff
@@ -17,6 +17,7 @@
 #include <string_view>
 #else
 #include <experimental/string_view>
+#define __STRING_VIEW_EXPERIMENTAL__
```

Comment:
It's better to use 'using' keyword:

```c++
#if __has_include(...)
#include <string_view>
using std::string_view;
#else
#include <experimental/string_view>
using std::experimental::string_view
#endif
```

Then you may remove namespace specifier below:
```c++
string_view name = {};
```

Reply:
Yeah, that would look a lot nicer. I'll change that tonight when i will be fixing the windows build.